src/frontend/apps/desk/src/features/teams/api/useTeams.tsx

```diff
@@ -34,7 +34,12 @@ export const getTeams = async ({
   const response = await fetchAPI(`teams/?page=${page}${orderingQuery}`);
 
   if (!response.ok) {
-    throw new Error(`Couldn't fetch teams: ${response.statusText}`);
+    const error: APIError = {
+      status: response.status,
+      causes: await errorCauses(response),
+    };
+
+    throw error;
```

Comment:
Would you need to update `useInfiniteQery` errors type?

Reply:
[86f5845](https://github.com/numerique-gouv/people/pull/52/commits/86f58452174e8c9394e6bd5c84ef0c7600ceed6b)